install/sdk/lib/SDKContainers.py

```diff
@@ -96,8 +109,11 @@ def get(
             docker.install_jumpscale(
                 force=False, reset=False, secret=secret, identity=identity, email=email, words=words,
             )
-
-            docker.executor.file_write("/sandbox/cfg/.configured", "")
+            if not "win32" in self.IT.MyEnv.platform():
```

Comment:
should be written as if "win32" not in

Reply:
used a variable in MyEnv instead of the check :)